### PR TITLE
Add  comments on client initialization and supported Version

### PR DESCRIPTION
**Pull request by shemogumbe**
Fixes https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/1937

## Overview

graph_client is currently referenced in snippets with no information on how to initialize it.

### Demo

before the line referencing graphcliet, a python comment is added saying

```python
# To initialize your graph_client, see https://learn.microsoft.com/en-us/graph/sdks/create-client?from=snippets&tabs=python

```
## Snippet Update to:
```python

# Code snippets are only available for the latest version. Current version is 1.x
from msgraph import GraphServiceClient
from msgraph.generated.users.users_request_builder import UsersRequestBuilder
from kiota_abstractions.base_request_configuration import RequestConfiguration
# To initialize your graph_client, see https://learn.microsoft.com/en-us/graph/sdks/create-client?from=snippets&tabs=python


query_params = UsersRequestBuilder.UsersRequestBuilderGetQueryParameters(
		filter = "accountEnabled eq false",
)

request_configuration = RequestConfiguration(
query_parameters = query_params,
)

result = await graph_client.users.get(request_configuration = request_configuration)

```

### CODE REVIEW

**Review by andrueastman — DISMISSED** [review]

File: CodeSnippetsReflection.OpenAPI/LanguageGenerators/PythonGenerator.cs [hunk 1]
```diff
@@ -67,13 +71,16 @@ public string GenerateCodeSnippet(SnippetModel snippetModel)
         {
             var indentManager = new IndentManager();
             var codeGraph = new SnippetCodeGraph(snippetModel);
-            var snippetBuilder = new StringBuilder($"{Environment.NewLine}{Environment.NewLine}" +
+            var snippetBuilder = new StringBuilder();
+            snippetBuilder.AppendLine(InitializationInfoString);
+            snippetBuilder.AppendLine($"{Environment.NewLine}{Environment.NewLine}" +
                                                    $"{ClientVarName} = {ClientVarType}({CredentialVarName}, {ScopesVarName}){Environment.NewLine}{Environment.NewLine}");
```
Comment (Ndiritu) [inline CodeSnippetsReflection.OpenAPI/LanguageGenerators/PythonGenerator.cs:77]: Should we remove the graph client instantiation line here?

Comment (andrueastman) [reply CodeSnippetsReflection.OpenAPI/LanguageGenerators/PythonGenerator.cs:77]: You're right. This should be dropped to match the spec.

File: CodeSnippetsReflection.OpenAPI/LanguageGenerators/PythonGenerator.cs [hunk 2]
```diff
@@ -67,13 +71,14 @@ public string GenerateCodeSnippet(SnippetModel snippetModel)
         {
             var indentManager = new IndentManager();
             var codeGraph = new SnippetCodeGraph(snippetModel);
-            var snippetBuilder = new StringBuilder($"{Environment.NewLine}{Environment.NewLine}" +
-                                                   $"{ClientVarName} = {ClientVarType}({CredentialVarName}, {ScopesVarName}){Environment.NewLine}{Environment.NewLine}");
```
Comment (andrueastman) [inline CodeSnippetsReflection.OpenAPI/LanguageGenerators/PythonGenerator.cs:71]: Couple of things
- Some of these variables are no longer used.
- Check the tests that are broken due to this change.

Comment (andrueastman) [reply CodeSnippetsReflection.OpenAPI/LanguageGenerators/PythonGenerator.cs:71]: Could we also remove the unused variables like `ClientVarName` and `ClientVarType` as well?

Comment (shemogumbe) [reply CodeSnippetsReflection.OpenAPI/LanguageGenerators/PythonGenerator.cs:71]: ClientVarName is referenced in the request execution path :
```
            string pathSegment = $"{ClientVarName}.{GetFluentApiPath(codeGraph.Nodes, codeGraph)}";
```


I have removed scopes, credentials and varType

**Review by andrueastman — APPROVED** [review]
This may also break raptor validation as it may expect the client initialization code to build/validate. 
We should probably confirm that before merging this. Otherwise, looks good from my end.

**Comment by shemogumbe** [comment]
> This may also break raptor validation as it may expect the client initialization code to build/validate. We should probably confirm that before merging this. Otherwise, looks good from my end.

Thought the same too, I will need to check if Raptor has a client initiated, if not, instatiate one within the snippets in Raptor, will keep this unmerged till I fix that

File: apidoctor
Comment (Ndiritu) [inline apidoctor:1]: nit; conflicts

Comment (Ndiritu) [inline apidoctor:1]: Ideally we shouldn't have changes to this submodule in this PR.

**Review by Ndiritu — CHANGES_REQUESTED** [review]

File: apidoctor
Comment (andrueastman) [reply apidoctor:1]: +1

**Review by Ndiritu — APPROVED** [review]

**Pull request merged**